Services/Description/ModDescriptionService.cs

```diff
@@ -28,6 +28,8 @@ namespace Coflnet.Sky.Api.Services;
 
 public class ModDescriptionService : IDisposable
 {
+    private static string BitsRegexPattern = @".*?(\d*\.?\d+|\d{1,3}(,\d{3})*(\.\d+)?) Bits.*";
```

Comment:
May be over engineered as Bits (like coins) prices seem to always be the only thing in the line so an `EndsWith(" Bits")` could do

Reply:
Yes, I was worried by it being triggered by different sources, because bits are mentioned in many places in Community shop ex. 
![image](https://github.com/Coflnet/SkyApi/assets/21691502/02217b3a-2b08-4429-bc02-5f0310eb4fa3)